clouddriver-alicloud/src/main/java/com/netflix/spinnaker/clouddriver/alicloud/model/AliCloudInstance.java

```diff
@@ -0,0 +1,73 @@
+package com.netflix.spinnaker.clouddriver.alicloud.model;
+
+import com.netflix.spinnaker.clouddriver.model.HealthState;
+import com.netflix.spinnaker.clouddriver.model.Instance;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author: luoguan
+ * @create: 2019-06-18 16:51
+ */
+public class AliCloudInstance implements Instance {
+
+  private String name;
+  private Long launchTime;
+  private String zone;
+  private String providerType;
+  private String cloudProvider;
+  private HealthState healthState;
+  private List<Map<String, Object>> health;
```

Comment:
If you have a better idea of the model for instance health, I would suggest more strongly-typing it instead of using a Map: We're trying to move past maps-of-maps across the project where possible.

Reply:
I think the instance health type is not decided by me.  It has been defined by the interface.

Reply:
Ah right... thanks. 👍